### ISSUE TITLE
documentation/process improvements 

### ISSUE THREAD

**Issue by TheTechnobear** [davidmoreno/rtpmidid#11]
some ideas of how to move forward... 

a) tagging/labels - in github
would be useful to be able to tag issues as bug/missing/features , and perhaps 'priority'
so we can get a feel for whats important, but still list things so they are not forgotten'

you can do this nicely with GitHub, by using 'setup issue templates',
when you add these, you can edit them, and automatically assign labels.
e.g. create BUG issue template , edit  and BUG label,  (repeat for Feature Request)
now users when they do new issue, will see the option to raise a bur or feature request, 
and its labelled correctly without you having to do the assignment (unless thats something you'd prefer.

b) tagging in code
use //TODO , and //FIXME in code? 
I find useful for code base when its still being developed, so from source code you know where (inevitable) shortcuts are taken during early stages ... also useful for documenting things that are a 'quick' implementation and might be subject to change later

c) debug/logging output
is there a warning level? 

-------

why im thinking of the above?

example:
Message Type 48 not implemented  (active sense) 

this gets repeated in logs... as debug

i think it would be good if this was instead a WARNING, and only printed the first time its encountered. (it shouldn't be debug, since when 'released' you'll still want to know about missing messages type - that we may have forgotten) 

GitHub - its not really a bug, just a missing feature

code tagging - 
it might be we just ignore active sense for now, or parse it but dont sent it out.
then it be nice to add a TODO/FIXME note in code to point this out.

**Comment by TheTechnobear** [comment]
(b) noticed there are TODO in code, so Ive started using - 
(c) & example  - implement in commit f5fcc68472c10f293cdfdfccc30afeae0086d7e0

(a) - GitHub labels  - for you to decide, if you want to do/manage yourself or let contributors do.
just let use know, then you can close this issue.

**Comment by davidmoreno** [comment]
I can see the issue tags, so if you can confirm they are there, we can close.

With PR #14 warn only once was also implemented for events. So I think is all fixed for this issue.
